app/forms/support/multiple_locations_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Support
+  class MultipleLocationsForm < Form
+    FIELDS = %i[
+      location_details
+    ].freeze
+
+    attr_accessor(*FIELDS)
+
+    validates :location_details, presence: true
+
+    def compute_fields
+      model.attributes.symbolize_keys.slice(*FIELDS).merge(new_attributes)
+    end
+
+    # will change in follow up PR
+    def form_store_key
```

Comment:
```suggestion
def form_store_key = :user
```

Ruby 3 sugar for one line methods

Reply:
Fixed: https://github.com/DFE-Digital/publish-teacher-training/pull/3398/commits/f69780d225a1760c5f5221154a6fea5eb0778ac7

I wonder if there's a cop for this 🤔